contracts/payloads/IGP15/PayloadIGP15.sol

```diff
@@ -0,0 +1,668 @@
+                address(this) == PROPOSER_AVO_MULTISIG,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Set market rates for sUSDe on Liquidity.
+        action1();
+
+        // Action 2: Set token config for sUSDe on Liquidity.
+        action2();
+
+        // Action 3: Deploy sUSDe/USDC vault and set related configs.
+        action3();
+
+        // Action 4: Deploy sUSDe/USDT vault and set related configs.
+        action4();
+
+        // Action 5: Update market rates for USDC.
+        action5();
+
+        // Action 6: Update market rates for USDT.
+        action6();
+
+        // Action 7: Update reward rates for USDC.
+        action7();
+
+        // Action 8: Update reward rates for USDT.
+        action8();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Set market rates for sUSDe on Liquidity.
+    function action1() internal {
+        AdminModuleStructs.RateDataV2Params[]
+            memory params_ = new AdminModuleStructs.RateDataV2Params[](1);
+
+        params_[0] = AdminModuleStructs.RateDataV2Params({
+            token: sUSDe_ADDRESS, // sUSDe
+            kink1: 50 * 1e2, // 50%
+            kink2: 80 * 1e2, // 80%
+            rateAtUtilizationZero: 0, // 0%
+            rateAtUtilizationKink1: 20 * 1e2, // 20%
+            rateAtUtilizationKink2: 40 * 1e2, // 40%
+            rateAtUtilizationMax: 100 * 1e2 // 100%
+        });
+
+        LIQUIDITY.updateRateDataV2s(params_);
+    }
+
+    /// @notice Action 2: Set token config for sUSDe on Liquidity.
+    function action2() internal {
+        AdminModuleStructs.TokenConfig[]
+            memory params_ = new AdminModuleStructs.TokenConfig[](1);
+
+        params_[0] = AdminModuleStructs.TokenConfig({
+            token: sUSDe_ADDRESS, // sUSDe
+            threshold: 0.3 * 1e2, // 0.3
```

Comment:
0.25 according to proposal (https://atlas.instadapp.io/proposals/15)

Reply:
Yes, should have been according to the proposal.